05_iigo/index.tex

```diff
@@ -407,9 +425,8 @@ \subsection{Budget}
 
 \subsection{Salary}
 \label{subsec:salary}
-A salary is paid to each role in power as an incentive to act in a publicly approved way. %Hence, each role has the \emph{power} to pay a salary to another role following the salary cycle in Figure~\ref{fig:cycles_in_IIGO}.
+A salary is paid to each role in power as an incentive to be in power. Since our system has regimented the means of power transfer to be an election, this incentive extends to acting in a publicly approved way. %Hence, each role has the \emph{power} to pay a salary to another role following the salary cycle in Figure~\ref{fig:cycles_in_IIGO}.
```

Comment:
DO NOT say power transfer.

say **transfer-of-power**

Reply:
the means of power transfer sounds better here. I disagree. I think it's ok to vary both mean the same thing at the end of the day.

Reply:
ok